docs/source/starter/installation.rst

```diff
@@ -114,7 +114,10 @@ This command installs all GeoIPS Python dependencies, and GeoIPS itself.
 .. code:: bash
 
     # Ensure geoips python environment enabled before installing geoips
-    pip install -e "$GEOIPS_PACKAGES_DIR/geoips"[doc,lint,test,debug]
+    # using "conda activate geoips"
+
+    # Install geoips via pip
+    pip install -e "$GEOIPS_PACKAGES_DIR/geoips[doc,lint,test,debug]"
```

Comment:
apparently this works on bash and zsh, the old way ONLY worked on bash.

Reply:
Yeah, on zsh it thinks that the `[doc,lint,test,debug]` part is a part of the filename. So it looks for `$GEOIPS_PACKAGES_DIR/geoips[doc,lint,test,debug]`. Moving the quote sends the whole thing to pip, instead, and lets pip parse that info.